components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/APIUsageStatisticsClient.java

```diff
@@ -152,14 +152,14 @@ public static OMElement buildOMElement(InputStream inputStream) throws Exception
         return builder.getDocumentElement();
     }
 
-    public List<APIResponseFaultCountDTO> getPerAppFaultCount(String subscriberName, String fromDate, String toDate, int limit)
+    public List<APIResponseFaultCountDTO> getPerAppFaultCount(String subscriberName, String groupId, String fromDate, String toDate, int limit)
```

Comment:
Shall we add method comments to all the missing places in this PR? (Line 155,193,249,311,399,439)